its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/MultiLanguageTest.java

```diff
@@ -398,7 +398,7 @@ public CreateGitFolder(Path projectDir) {
     public void commitAll() {
       try (var git = Git.open(gitDir.toFile())) {
         git.add().addFilepattern(".").call();
-        git.commit().setMessage("Initial commit").call();
+        git.commit().setMessage("Initial commit").setSign(false).call();
```

Comment:
I have git configured to sign all commits. This failed for some reason and I just turned it off as it isn't needed at all (The git commit is only making sure, that the sonar version control file filter is doing his job and filters .gitignored files).